### PR TITLE
[IMPROVEMENT-8178] Add Netty processor in Spring container

### LINKED ISSUE #8178: [Improvement][Master] Add the processor in Spring container
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

We use several Netty processors in Master and Worker. 
Right now, some are config by Spring container, and some are not config by Spring, in order to make the code consistent, it' better to add all processor into Spring container.

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ruanwenjun**
close #8178 

## Purpose of the pull request
To make the code style consistent.

## Brief change log
Make all processor as Spring bean.

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/AlertClientService.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AlertClientService {
+public class AlertClientService implements AutoCloseable {
```
Comment (yimaixinchen) [inline dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/AlertClientService.java:31]: does it add component annotation?

Comment (kezhenxu94) [reply dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/AlertClientService.java:31]: > does it add component annotation?

This component depends on something that are not inside this module, so we have to initialize it in the module where needed, check `dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/config/BeanConfig.java`

**Review by kezhenxu94 — DISMISSED** [review]
LGTM if tests passed

**Review by caishunfeng — APPROVED** [review]
LGTM

**Pull request merged**